### PR TITLE
Invalidate entry's sheet collection when delete

### PR DESCRIPTION
**Pull request by BeckyPollard**
## Summary of changes:

- Add `invalidates` link for the `entry` delete controller action thing.
  - Why: when deleting an entry, on the frontend the collection items would still maintain the deleted entity as part of the rendered list. Now, the collection updates on delete and the render is correct.

----
Feedback, nitpicks, and/or corrections encouraged. 🤖

### CODE REVIEW

File: src/entry/controller/item.ts [hunk 1]
```diff
@@ -6,39 +6,28 @@ import * as entryService from '../service';
...
-
     ctx.response.type = 'application/hal+json';
 
-    const person = await personService.findById(
-      +ctx.params.personId,
-    );
+    const person = await personService.findById(+ctx.params.personId);
 
     ctx.response.body = hal.item(
-      await entryService.findById(
```
Comment (evert) [inline src/entry/controller/item.ts:23]: I noticed your PRs are doing a lot of unrelated changes, which makes it a bit hard to figure out what was actually changed. I assume this is some kind of auto-formatter?

Comment (BeckyPollard) [reply src/entry/controller/item.ts:23]: **oh gosh** yes I didn't catch that at all this time, have been using Prettier since working on the previous client

Comment (evert) [reply src/entry/controller/item.ts:23]: Makes sense. If you love prettier, i think it's better to apply it wholesale in a separate PR. If you can take it or leave it, I'd vote for no prettier ;)

Comment (BeckyPollard) [reply src/entry/controller/item.ts:23]: I think no prettier is better in this case 100%

**Review by evert — APPROVED** [review]
